src/apps/bridge/reportBuilder.cpp

```diff
@@ -88,6 +92,7 @@ typedef struct ReportBuilderContext_s {
   uint32_t _transmitAggregations;
   ReportBuilderLinkedList _reportBuilderLinkedList;
   uint64_t _report_period_node_list[TOPOLOGY_SAMPLER_MAX_NODE_LIST_SIZE];
+  abstractSensorType_e _report_period_sensor_type_list[TOPOLOGY_SAMPLER_MAX_NODE_LIST_SIZE];
```

Comment:
use a list to map a node_id to an abstract sensor type enum value. This is to allow us to add a NAN report for a sensor that may be part of our topology + crc, but for some reason was not subscribed to in the sensorController.